src/ResourceManager/StorageSync/Commands.StorageSync/help/Invoke-AzureRmStorageSyncCompatibilityCheck.md

```diff
@@ -0,0 +1,170 @@
+- Maximum dataset size
+- Maximum directory depth
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Invoke-AzureRmStorageSyncCompatibilityCheck C:\DATA
+```
+
+This command checks the compatibility of the system and also of files and folders in C:\DATA.
+
+### Example 2
+```powershell
+PS C:\> Invoke-AzureRmStorageSyncCompatibilityCheck C:\DATA -SkipSystemChecks
+```
+
+This command checks the compatibility of files and folders in C:\DATA, but does not perform a system compatibility check.
+
+### Example 3
+```powershell
+PS C:\> $errors = Invoke-AzureRmStorageSyncCompatibilityCheck C:\DATA
+PS C:\> $errors | Select-Object -Property Type, Path, Level, Description, Result | Export-Csv -Path C:\results 
+```
+
+This command checks the compatibility of the system and also of files and folders in C:\DATA, and then exports the results as a CSV file to C:\results.
+
+## PARAMETERS
+
+### -ComputerName
+The computer you are performing this check on.
+
+```yaml
+Type: String
+Parameter Sets: ComputerNameBased
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Credential
+Your credentials for the share you are validating.
+
+```yaml
+Type: PSCredential
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Path
+The UNC path of the share you are validating.
+
+```yaml
+Type: String
+Parameter Sets: PathBased
+Aliases:
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Quiet
+Suppresses writing output report to console.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -SkipNamespaceChecks
+Set this flag to skip file namespace validations and only perform system validations.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: PathBased
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -SkipSystemChecks
+Set this flag to skip system validations and only perform file namespace validations.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+### None
+This cmdlet does not accept any input.
+
+## OUTPUTS
+
+### System.Object
```

Comment:
@afedyashov this should be `PSValidationResult`

Reply:
Thanks, fixed